dask_distance/_utils.py

```diff
@@ -0,0 +1,31 @@
+import dask
+import dask.array
+
+import dask_distance._compat
+
+
+def _bool_cmp_mtx_cnt(u, v):
+    u = dask_distance._compat._asarray(u)
+    v = dask_distance._compat._asarray(v)
+
+    u_1 = u.astype(bool)
+    v_1 = v.astype(bool)
+    u_0 = ~u_1
+    v_0 = ~v_1
+
+    uv_11 = u_1 & v_1
+    uv_10 = u_1 & v_0
+    uv_01 = u_0 & v_1
+    uv_00 = u_0 & v_0
+
+    uv_11_sum = uv_11.sum(axis=0, dtype=float)
+    uv_10_sum = uv_10.sum(axis=0, dtype=float)
+    uv_01_sum = uv_01.sum(axis=0, dtype=float)
+    uv_00_sum = uv_00.sum(axis=0, dtype=float)
```

Comment:
All this copy pasta is a bit bulky and error prone. Would be good if we could roll these up in `for`-loops or something. Don't need to worry about it now. However if we start mucking with this code more, we might want to give it some thought.